com.unity.render-pipelines.core/Editor/CoreEditorDrawers.cs

```diff
@@ -554,9 +571,12 @@ void IDrawer.Draw(TData data, Editor o)
                 else
                 {
                     CoreEditorUtils.DrawSplitter(isBoxed);
-                    newExpended = CoreEditorUtils.DrawHeaderFoldout(title, expended, isBoxed,
+                    newExpended = CoreEditorUtils.DrawHeaderFoldout(title,
+                        expended,
```

Comment:
could we rename with an 'a' ? should probably be expanded and not expended :)

Reply:
doesnt have to be this PR :)

Reply:
I will do :)